lib/src/pages/call.dart

```diff
@@ -52,13 +52,15 @@ class _CallPageState extends State<CallPage> {
     await AgoraRtcEngine.enableWebSdkInteroperability(true);
     await AgoraRtcEngine.setParameters(
         '''{\"che.video.lowBitRateStreamParameter\":{\"width\":320,\"height\":180,\"frameRate\":15,\"bitRate\":140}}''');
-    await AgoraRtcEngine.joinChannel(null, widget.channelName, null, 0);
+    await AgoraRtcEngine.joinChannel(null, widget.channelName, null, 1);
```

Comment:
the userId of the broadcaster is better a property or a static constant, to make developers understand.

Reply:
My bad.. i was actually just testing... im totally screwed up suppose to pull request to the cloned project... my bad..

Reply:
but noted thanks for the advice